bee-pow/src/score.rs

```diff
@@ -0,0 +1,46 @@
+// Copyright 2020 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! Contains utilities to score Proof of Work.
+
+use bee_crypto::ternary::{
+    sponge::{CurlP81, Sponge},
+    HASH_LENGTH,
+};
+use bee_ternary::{b1t6, Btrit, T1B1Buf, TritBuf, Trits, T1B1};
+
+use crypto::hashes::{blake2b::Blake2b256, Digest};
+
+/// Computes the Proof of Work score of given bytes.
+pub fn compute_pow_score(bytes: &[u8]) -> f64 {
+    pow_score(&pow_hash(bytes), bytes.len())
+}
+
+/// Returns the Proof of Work hash of given bytes.
+pub fn pow_hash(bytes: &[u8]) -> TritBuf<T1B1Buf> {
+    assert!(bytes.len() >= std::mem::size_of::<u8>());
```

Comment:
if we don't control the bytes length, then it's probably better to return a `Result`, rather than panicking here. If we do, then `debug_assert!` should be enough I think.